packager/media/formats/mp4/segmenter.cc

```diff
@@ -216,7 +180,14 @@ Status Segmenter::FinalizeSegment(size_t stream_id,
           {key_frame_info.timestamp, moof_start_offset,
            fragment_buffer_->Size() - moof_start_offset + key_frame_info.size});
```

Comment:
Also, is this math still correct? Seems like moof_start_offset = fragment_buffer_->Size() , which would make this = 
key_frame_info.size.

Reply:
Thanks for this catch! The math was actually off here, but in a different way. The original calculation in pseudo code is essentially (moof size + mdat header size - moof offset + key frame size). I reworked the PR to match this. The fragment buffer `fragmenter->fragment()->Size()` contains (moof size + mdat header size + data size), so I am now subtracting the data buffer `fragmenter->data()->Size()`, which contains ALL of the frames within the fragment, from this to isolate the header info sizes. I just verified that the metadata here matches the output for the Packager master branch.

Reply:
Perfect!!